src/main/java/com/hms/flexyosisoftconnector/dataserver/OSIsoftServer.java

```diff
@@ -76,6 +79,25 @@ public class OSIsoftServer {
   /** Constructs the OSIsoftServer object. */
   public OSIsoftServer() {}
 
+  /** HTTP Method strings */
+  static final String HTTP_METHOD_POST = "POST";
+
+  static final String HTTP_METHOD_GET = "GET";
+
+  /**
+   * Repeated Exception handling code for HTTP requests
+   *
+   * @param e Exception
+   * @param ErrorMessage Specific message to log
+   * @return error code
+   */
+  private static int RequestHTTPSError(Exception e, String ErrorMessage) {

```

Comment:
```suggestion
  private static int RequestHttpsError(Exception e, String ErrorMessage) {
```
Google Java Style Guide does not allow for acronyms to be capitalized. See https://google.github.io/styleguide/javaguide.html#s5.3-camel-case